pages/index.tsx

```diff
@@ -126,7 +127,7 @@ const Index = () => {
         } else if (tipo === "4") {
             return 10
         } else if (tipo === "5") {
-            return 70
+            return 36
```

Comment:
I'm 100% ok for this change, with a minor caveat: where in the Pix spec is this specified? As far as I remember, "Random Keys" (aka `Chave Aleatória`) can have up to 70 characters.

The current common practice is to assign a UUID as such keys, but a SHA could be used as well.

What do you think?

My ideas on how to go about this:

1. **Adding a new type:** The old type called **Chave Aleatória** would be a UUID as implemented here. The new type can be something like **Chave Aleatória (Não Padrão)** and would be simply a 70 character, alphanumerical field.
2. **Changing it like it is and...** well, I'm sure we'll know if the standard changes 😅